docs/BackupRDB.md

```diff
@@ -0,0 +1,14 @@
+# Backup RDB guide
+
+This guide explains how to backup single RDB data with Scalar products on a Kubernetes environment. We assume that you use managed databases provided by cloud service.
```

Comment:
```suggestion
This guide explains how to create a backup of a single relational database (RDB) by using ScalarDB or ScalarDL in a Kubernetes environment. Please note that this guide assumes that you are using managed databases provided by a cloud service.
```

Reply:
As I mentioned in another comment, I think it would be better to describe `databases that are used by ScalarDB or ScalarDL` instead of `databases that use ScalarDB or ScalarDL`.

What do you think?